### DISCUSSION TITLE
When doing a complete download of my games and extras, DLCs are not included

### DISCUSSION THREAD

**Discussion by bfaurot** [Kalanyr/gogrepo#107] [Q&A] [answered]
When doing a complete download of my games and extras, DLCs are not included.  The manuals and soundtracks and other extras download.  I don't see any mention of this in the documentation.  Am I using the script incorrectly or, is this not an included feature in the script?

**Comment by Kalanyr** [comment]
DLC is usually considered an Installer rather than an Extra and so will
generally be with the installation files.

**Reply by bfaurot** [reply]
Thank you for the response.  Once I knew what was supposed to occur, I tested updating one ```id``` and it worked.  It was skipped when just doing an update with no specificity.  I need to recreate my manifest file occasionally.

**Comment by cyotek** [comment, marked as answer]
Not sure if this is the case here, but there is one scenario I come across where DLCS do end up missing - if you buy the DLC _after_ you've previously done an `update` for the base game. This command detects changes to game metadata, but doesn't seem to detect DLC purchases. So after I've bought DLC, I usually issue a separate `update` with `-ids` to force a full update of that particular game which will then update the manifest with the DLC data, followed by the usual `download`, `clean` and `verify` to pull down the new files.

Once in a blue moon I also start from scratch with `update -full` just in case I've forgotten to update a particular title or just to correct any other errors that creep in. But as that takes an awfully long time given the size of my GoG library it is something I only do once or twice a year.

As I said, not sure if this the case, but perhaps the info will help!

**Reply by bfaurot** [reply]
Thank you both for your replies.  This was the case.  When I updated using the specific id the dlcs were added to the manifest.  I need to recreate the manifest to catch all my dlc purchases.
